CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ if(POLICY CMP0135)
 endif()
 
 set(CMAKE_OSX_DEPLOYMENT_TARGET "10.15" CACHE STRING "Minimum macOS / OS X deployment version")
+set(CMAKE_SYSTEM_VERSION 10.0.22621.0)
```

Comment:
Can I ask what this does?

Reply:
It sets the minimal windows sdk to target for builds. The minimal version I added is the first one that includes all the Blend enum values.

Reply:
Can we add a comment here? THhis will probably need to be moved when we plugin-ify things.